l10n/en/brands.ftl

```diff
@@ -186,3 +186,7 @@
 -brand-name-google-play = Google Play
 -brand-name-youtube = YouTube
 -brand-name-chromebook = Chromebook
+
+## Websites
+
+-brand-name-hibp = Have I Been Pwned
```

Comment:
The latest guidance from the L10n team is that we should only use these brand name placeholders for our own products going forward, so `Have I Been Pwned` can just be a regular string.